connectors/sources/onedrive.py

```diff
@@ -581,6 +581,7 @@ def prepare_doc(self, file):
             "created_at": file.get("createdDateTime"),
             "size": file.get("size"),
             "url": file.get("webUrl"),
+            "mime_type": file.get("file", {}).get("mimeType"),
```

Comment:
This can throw an error if `file` is `None`:

```
>>> test = { "file": None }
>>> test.get("file", {})
>>> test.get("file", {}).get("mimeType")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'get'
```

Can you change it to cover this edge case?

Reply:
Also could you add a test to cover if `file` is `None`?